test/correctness/specialize.cpp

```diff
@@ -394,7 +394,10 @@ int main(int argc, char **argv) {
         out.compute_root().specialize(cond1 && cond2).vectorize(x, 4);
 
         if_then_else_count = 0;
-        out.add_custom_lowering_pass(new CountIfThenElse());
+        CountIfThenElse *pass1 = new CountIfThenElse();
```

Comment:
Now that this isn't being handed off to a custom lowering pass, it should probably just be a stack variable:

    CountIfThenElse pass1;
    pass1.mutate(...);
    ...